browser/about_flags.cc

```diff
@@ -368,6 +373,11 @@ constexpr char kAllowCertainClientHintsDescription[] =
     "Allows setting certain request client hints (sec-ch-ua, sec-ch-ua-mobile, "
     "sec-ch-ua-platform)";
 
+#if defined(TOOLKIT_VIEWS)
+constexpr char kBraveVerticalTabsName[] = "Vertical tabs";
+constexpr char kBraveVerticalTabsDescription[] =
+    "Uses vertical tabs instead of tabstrip";
```

Comment:
Well, I guess vertical or horizontal are still "strips" 😄 
Perhaps the description can be something like "Move tab strip to be a vertical panel on the side of the window instead of horizontal at the top of the window."?

Reply:
That's better. Thank!